src/uct/sm/mm/base/mm_iface.c

```diff
@@ -324,7 +326,7 @@ static unsigned uct_mm_iface_progress(uct_iface_h tl_iface)
     unsigned total_count  = 0;
     unsigned count;
 
-    ucs_assert(iface->fifo_wnd >= UCT_MM_IFACE_FIFO_MIN_WINDOW);
+    ucs_assert(iface->fifo_wnd >= UCT_MM_IFACE_FIFO_MIN_POLL);
```

Comment:
maybe rename fifo_wnd to fifo_poll_count?

Reply:
done